docs-new/docs/components/certificate-operator.md

```diff
@@ -1,65 +1,23 @@
-# Keptn Certificate Manager
+# Keptn Cert Manager
 
-## Keptn Cert Manager
-
-The Keptn Cert Manager automatically configures TLS certificates to
+The Keptn Cert Manager is a Kubernetes operator that
+automatically configures TLS certificates to
 [secure communication with the Kubernetes API](https://kubernetes.io/docs/concepts/security/controlling-access/#transport-security).
-You can instead
-[use cert-manager.io](../installation/configuration/cert-manager.md)
-for this purpose.
 
-Keptn includes a Mutating Webhook
-that requires TLS certificates to be mounted as a volume in its pod.
-In version 0.6.0 and later, the certificate creation
-is handled automatically by
-the [keptn-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/keptn-cert-manager/README.md).
+Keptn includes a Mutating, Validating and Conversion Webhooks
+that requires TLS certificates to be mounted as a volume.
+These webhooks and CRDs contain a `keptn.sh/inject-cert: "true"` label that indicates that
+these resources require a TLS certificate.
 
 How it works:
 
-* The certificate is created as a secret
-in the `keptn-system` namespace
-with a renewal threshold of 12 hours.
-* If the certificate expires,
-the [keptn-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/keptn-cert-manager/README.md)
-renews it.
-* The Keptn `lifecycle-operator` waits for a valid certificate to be ready.
-* When the certificate is ready,
-  it is mounted on an empty dir volume in the operator.
-
-`keptn-cert-manager` is a customized certificate manager
-that is installed with Keptn by default.
-It is included to simplify installation for new users
-and because it is much smaller than most standard certificate managers.
-However, Keptn is compatible with most certificate managers
-and can be configured to use another certificate manager if you prefer.
-See [Use Keptn with cert-manager.io](../installation/configuration/cert-manager.md)
-for instructions.
-
-## Invalid certificate errors
```

Comment:
Do we really want to delete all this information?  It looks quite useful to me for troubleshooting.  I suppose it could be moved to a page in "Operations" or "Troubleshooting" if we had such a section.

Reply:
I would suggest to link a troubleshooting docs of the cert-manager.io https://cert-manager.io/docs/troubleshooting/

as this is the reference which describes most of the problems that can happen. In this section we are describing only one of the problems, so I would say let's make our life easier :)

What do you think?

Reply:
I suppose that could work.  I prefer to have depth in the architectural descriptions and have the troubleshooting docs be more like how-to lists that link to the architecture docs for the background but Mo seems to have decded that the architecture information should be high-level.

Meanwhile, I hate to have this information deleted before it is transferred someplace or at least copied into a new issue that describes where to put it..